l10n/de_DE/LC_MESSAGES/remoteevent.po

```diff
@@ -1349,8 +1418,70 @@ msgstr ""
 
 #: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
 msgid ""
-"If you tick this box, registrations will be assigned the \"Awaiting approval"
-"\" status and you will need to manually review and approve each "
+"Whether participants can register with themselves up to 9 other participants "
+"using the profile to be selected below."
+msgstr ""
+"Bestimmt, ob Veranstaltungsteilnehmer mit ihrer Anmeldung bis zu 9 weitere "
+"Teilnehmer anmelden können, unter Verwendung des unterhalb ausgewählten "
+"Profils."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid "The registration profile to use for additional participants."
+msgstr "Zur Anmeldung zusätzlicher Teilnehmer verwendetes Anmeldeprofil."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"Limit the number of additional participants that can be registered in a "
+"single booking."
+msgstr ""
+"Beschränkung der Anzahl zusätzlicher Teilnehmer, die in einer "
+"Einzelanmeldung mit angemeldet werden können."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"Eg: if you choose '2' then the lead booker can bring 2 guests; there would "
+"be a limit of 3 participants in total per booking."
+msgstr ""
+"Wenn Sie z. B. '2' auswählen, kann der Teilnehmer bis zu 2 Gäste mitbringen; "
+"Es gäbe eine Beschränkung von insgesamt 3 Teilnehmern pro Buchung."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"Whether to allow additional participants be added to a waiting list (if "
+"active) in case there are not enough vacancies for the total number of "
+"participants requested to be registered."
+msgstr ""
+"Bestimmt, ob zusätzliche Teilnehmer zu einer ggf. aktiven Warteliste "
+"hinzugefügt werden sollen, falls nicht genügend freie Plätze für die Anzahl "
+"zusätzlich anzumeldender Teilnehmer vorhanden sind."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"When this is active, some additional participants might be added to the "
+"waiting list instead of being registered."
+msgstr ""
+"Wenn aktiviert, könnten einige zusätzliche Teilnehmer zur Warteliste "
+"hinzugefügt werden, statt direkt angemeldet zu werden."
+
+#: templates/CRM/Remoteevent/Form/RegistrationConfig.hlp
+msgid ""
+"The XCM tool will be used to identify and update an existing contact with "
+"the submitted data, or to create a new one."
+msgstr ""
+"Das XCM-Werkzeug wird verwendet, um bestehende Kontakte zu identifizieren "
+"und mit übermittelten Daten zu aktualisieren, oder um einen neuen Kontakt "
```

Comment:
I think there should be no `,` before `oder`.

Reply:
That's actually on purpose - with "new" German orthography, commata are still allowed, and to be used for structuring complex phrases. The *oder* introduces a separate subclause that is semantically separate from the previous one (which is itself a compound of two subclauses, combined with *und*) and is thus explicitly separated by the comma.